src/main/java/com/example/nasapicturesstealer/configuration/NasaConfig.java

```diff
@@ -6,7 +6,6 @@
 
 @Configuration
 public class NasaConfig {
-
     @Bean
     public ObjectMapper objectMapper(){
```

Comment:
you don't need to create ObjectMapper instance here, the spring boot web starter dependency creates this bean by default